packages/@lwc/engine/src/framework/membrane.ts

```diff
@@ -7,6 +7,8 @@
 import ObservableMembrane from 'observable-membrane';
 import { valueObserved, valueMutated } from './mutation-tracker';
 
+export const membraneTag = Symbol.for('@@lockerLiveValue');
```

Comment:
@ravijayaramappa has confirmed that indeed we have a polyfill for `Symbol.for` in IE11 today.